### ISSUE TITLE
Divine Aegis

### ISSUE THREAD

**Issue by LocalWyrdo** [dalaranwow/dalaran-wow#4879]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

The talent Divine Aegis in the Discipline tree (Priest) only procs on self heals.

**Expected behaviour**: Tell us what should happen instead

Every heal target should get the shield, not just the Priest.

**Steps to reproduce the problem**:

1. Crit a heal on self: talent works.
2. Crit a heal on someone else: nothing happens.
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by StheNoble** [comment]
i cant confirm this bug. Devine aegis works perfectly fine and your reports contains no proof whatsoever.

here is a current combatlog report from dala wow from a disc priest. Check the tab "healing by spell"
http://www.worldoflogs.com/reports/rt-eylfz2nz9qylkigg/details/7/
